docs/docsite/rst/vmware/scenario_remove_vm.rst

```diff
@@ -0,0 +1,127 @@
+.. _vmware_guest_remove_virtual_machine:
+
+*****************************************
+Remove an existing VMware virtual machine
+*****************************************
+
+.. contents:: Topics
+
+Introduction
+============
+
+This guide will show you how to utilize Ansible to remove an existing VMware virtual machine.
+
+Scenario Requirements
+=====================
+
+* Software
+
+    * Ansible 2.5 or later must be installed.
+
+    * The Python module ``Pyvmomi`` must be installed on the Ansible (or Target host if not executing against localhost).
```

Comment:
I'm confused by this. If I'm spinning down a VMware VM, wouldn't I always be executing against localhost? Is it common to run Ansible on one machine and execute on another to remove a VM?

Also, I'd change "installed on the Ansible" to "installed on the Ansible control node".

Reply:
We have description at "Example description" so this OK for now.

Reply:
Sounds good. I made my comments as I read, instead of reading through the whole document and then commenting. So now I understand why a user might not be executing on localhost.